plasma_framework/contracts/src/vaults/Vault.sol

```diff
@@ -69,4 +68,18 @@ contract Vault is Operated {
             return depositVerifiers[0];
         }
     }
+
+    /**
+     * @dev Function that should be called by the child contract inheritancing this on deposit.
```

Comment:
What does that mean?

Reply:
Is this better? 
https://github.com/omisego/plasma-contracts/pull/325/commits/ea33eea626f932f261d6b77c4b3faa529d7d3584

Reply:
`@dev designed to be called by the contract that inherits Vault`